apps/docs/content/guides/platform/advanced-log-filtering.mdx

```diff
@@ -0,0 +1,193 @@
+
+![Expanding log results](/docs/img/guides/platform/expanded-log-results.png)
+
+## Filtering with [regular expressions](https://en.wikipedia.org/wiki/Regular_expression)
+
+The Logs use BigQuery Style regular expressions with the [regexp_contains function](https://cloud.google.com/bigquery/docs/reference/standard-sql/string_functions#regexp_contains). In its most basic form, it will check if a string is present in a specified column.
+
+```sql
+select
+  cast(timestamp as datetime) as timestamp,
+  event_message,
+  metadata
+from postgres_logs
+where regexp_contains(event_message, 'is present');
+```
+
+There are multiple filters that you should consider using:
+
+### Find messages that start with a phrase
+
+`^` only looks for values at the start of a string
+
+```sql
+-- find only messages that start with connection
+regexp_contains(event_message, '^connection')
+```
+
+### Find messages that end with a phrase:
+
+`$` only looks for values at the end of the string
+
+```sql
+-- find only messages that ends with port=12345
+regexp_contains(event_message, '$port=12345')
+```
+
+### Ignore case sensitivity:
+
+`(?i)` ignores capitalization for all proceeding characters
+
+```sql
+-- find all event_messages with the word "connection"
+regexp_contains(event_message, '(?i)COnnecTion')
+```
+
+### Wildcards:
+
+`.` can represent any string of characters
+
+```sql
+-- find event_messages like "hello<anything>world"
+regexp_contains(event_message, 'hello.world')
+```
+
+### Alphanumeric ranges:
+
+`[1-9a-zA-Z]` finds any strings with only numbers and letters
+
+```sql
+-- find event_messages that contain a number between 1 and 5 (inclusive)
+regexp_contains(event_message, '[1-5]')
+```
+
+### Repeated values:
+
+`x*` zero or more x
+`x+` one or more x
+`x?` zero or one x
+`x{4,}` four or more x
+`x{3}` exactly 3 x
+
+```sql
+-- find event_messages that contains any sequence of 3 digits
+regexp_contains(event_message, '[0-9]{3}')
+```
+
+### Escaping reserved characters:
+
+`\.` interpreted as period `.` instead of as a wildcard
+
+```sql
+-- escapes .
+regexp_contains(event_message, 'hello world\.')
+```
+
+### `or` statements:
+
+`x|y` any string with `x` or `y` present
+
+```sql
+-- find event_messages that have the word 'started' followed by either the word "host" or "authenticated"
+regexp_contains(event_message, 'started host|authenticated')
+```
+
+### and/or/not statements in SQL:
+
+`and`, `or`, and `not` are all native terms in SQL and can be used in conjunction with regular expressions to filter results
+
+```sql
+select
+  cast(timestamp as datetime) as timestamp,
+  event_message,
+  metadata
+from postgres_logs
+where
+  (regexp_contains(event_message, 'connection') and regexp_contains(event_message, 'host'))
+  or not regexp_contains(event_message, 'received');
+```
+
+### Filtering and unnesting example
+
+**Filter for Postgres**
```

Comment:
I think a few different examples for each table would be good, perhaps with tabs.

Also can make a reference to the templates available.